meta-facebook/yv35-cl/src/ipmi/plat_ipmi.c

```diff
@@ -1064,3 +1065,23 @@ void pal_OEM_I2C_DEV_SCAN(ipmi_msg *msg) {
   msg->completion_code = CC_SUCCESS;
   return;
 }
+
+void pal_OEM_GET_POST_CODE(ipmi_msg *msg) {
+  uint8_t i;
```

Comment:
Is this variable used anywhere?
If not could you please remove it.

Reply:
OK, we will remove it and PR again.